tsconfig.json

```diff
@@ -13,10 +13,15 @@
     "incremental": true,
     "skipLibCheck": true,
     "strictNullChecks": false,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
Why? Akurat to dosyć istota rzecz.

Reply:
Moze dlatego ze na początku przeszkadza, ale raczej nie ma miejsca gdzie jesta any, chba ze przez niedopatrzenie😉